owslib/ogcapi/edr.py

```diff
@@ -0,0 +1,98 @@
+# =============================================================================
+# Copyright (c) 2023 Tom Kralidis
+#
+# Author: Tom Kralidis <tomkralidis@gmail.com>
+#
+# Contact email: tomkralidis@gmail.com
+# =============================================================================
+
+from io import BytesIO
+import logging
+from typing import BinaryIO
+
+from owslib.ogcapi.features import Features
+from owslib.util import Authentication
+
+LOGGER = logging.getLogger(__name__)
+
+
+class EnvironmentalDataRetrieval(Features):
+    """Abstraction for OGC API - Envirionmental Data Retrieval"""
+
+    def __init__(self, url: str, json_: str = None, timeout: int = 30,
+                 headers: dict = None, auth: Authentication = None):
+        __doc__ = Features.__doc__  # noqa
```

Comment:
Isn't there a need for a custom docstring ?

Reply:
Not at this level, given the building blocks of OGC APIs (collections are the same across APIS). docstrings are more specific deeper into the endpoints.